### ISSUE TITLE
russian support

### ISSUE THREAD

**Issue by Knoot** [zaldor30/GuildRecruiter#91]
**Describe the bug**
Currently the search does not work at all for the Russian version.

**To Reproduce**
1. change game language to russian
2. start search
3. See nothing

**Expected behavior**
Players must be found

**Additional context**
The problem is in the keys for the filter. Filter by race, Filter by race, for Russians it is "р", not "r".
Filter by class, for Russians it is "к", not "c".
Potential filter by zone, for Russians it is "з", not "z".
"р"/"к"/"з" - are not Latin characters

**Comment by zaldor30** [comment]
I believe this is an Ace3 issue.  I just downloaded the new Ace3 libraries and replaced the ones I am using.  That control was made in Ace3, let me know if that fixed it.

**Comment by Knoot** [comment]
No, it didn't help. The problem is in screens\GR_Scanner.lua:624 and :641
`local query = 'c-"'..v.name..'"'`
`local query = 'r-"'..v.name..'"'`
"c", "r" and potential "z" must be localized

**Comment by zaldor30** [comment]
I am doing that.  The top is the info I am using where the upper/lower case is the v.name and the all upper is the classFile.
![Wow_KQDiTgGXZo](https://github.com/user-attachments/assets/0b8644c0-cf92-46b8-b672-c6cc5f672fbc)

The mixed case name is the local version.

**Comment by zaldor30** [comment]
The big box
![Wow_XuVudL8duT](https://github.com/user-attachments/assets/8e756cf2-5a6f-4ddc-9db1-c31ae8ff4590)

**Comment by Knoot** [comment]
localized class name is used, no problem here. Same with races. Note the filter name  
Correct ("к-")
![image](https://github.com/user-attachments/assets/f89772c1-877d-4205-a63a-5d5deb8e2778)
Incorrect ("c-") the addon makes a request like this
![image](https://github.com/user-attachments/assets/d112b035-4bd2-478b-aeb0-a3b4de845555)
Воин = Warrior

**Comment by zaldor30** [comment]
Is that a specific filter rule for Russian?  I was going off the wowpedia using C_FriendList.SendWho('bob z-"Teldrassil" r-"Night Elf" c-"Rogue" 10-15') format as the ruleset.  I don't have any localization set up for Russian at this time.  I have Ace localization in place, but no translations.  But, did not know that the /who command changes for other languages.

**Comment by zaldor30** [comment]
Ok, I see what you are saying now.  
з = zone (зона)
р = race (раса)
к = class (класс)

That is needed instead of the z- r- c-.  Is there a good way to get system messages in the language of the interface without having to change the interface to that language?  I can't change it past Spanish and I rely on those messages.

**Comment by Knoot** [comment]
I only know one way - changing the interface language.
For FGI I added these filters to the localization on curseforge. It is likely that these filters should only change for Russian, and possibly for Chinese
